### ISSUE TITLE
[BUG] CRC high CPU and disk IO with minimum memory allocation ( 9GB)

### ISSUE THREAD

**Issue by charlesxucheng** [code-ready/crc#2808]
### General information

  * OS: Windows
  * Hypervisor:  Hyper-V
  * Did you run `crc setup` before starting it ? Yes
  * Running CRC on: Laptop

## CRC version
```bash
CodeReady Containers version: 1.33.1+3cfaa10f
OpenShift version: 4.8.12 (bundle installed at C:\Program Files\CodeReady Containers\crc_hyperv_4.8.12.crcbundle)
```
  
## CRC status
```bash
time="2021-10-20T07:58:40+08:00" level=warning msg="Failed to add proxy CA to crc http transport"
DEBU CodeReady Containers version: 1.33.1+3cfaa10f
DEBU OpenShift version: 4.8.12 (bundle installed at C:\Program Files\CodeReady Containers\crc_hyperv_4.8.12.crcbundle)
DEBU Running 'crc status'
DEBU Checking file: <User Home>\.crc\machines\crc\.crc-exist
DEBU Checking file: <User Home>\.crc\machines\crc\.crc-exist
DEBU Running 'Hyper-V\Get-VM crc | Select-Object -ExpandProperty State'
DEBU Running SSH command: df -B1 --output=size,used,target /sysroot | tail -1
DEBU Using ssh private keys: [<User Home>\.crc\machines\crc\id_ecdsa <User Home>\.crc\cache\crc_hyperv_4.8.12\id_ecdsa_crc]
DEBU SSH command results: err: <nil>, output: 32737570816 20793249792 /sysroot
DEBU Unexpected operator status for etcd: RecentBackup
CRC VM:          Running
OpenShift:       Running (v4.8.12)
Disk Usage:      20.79GB of 32.74GB (Inside the CRC VM)
Cache Usage:     28.48GB
Cache Directory: <User Home>\.crc\cache
```

## CRC config
```bash
time="2021-10-20T07:53:48+08:00" level=warning msg="Failed to add proxy CA to crc http transport"
- consent-telemetry                     : no
- host-network-access                   : true
- http-proxy                            : http://mylaptophostname:3128
- https-proxy                           : http://mylaptophostname:3128
- no-proxy                              :  localhost,127.0.0.1,10.96.0.0/12,192.168.99.0/24,192.168.39.0/24,192.168.50.0/24
- proxy-ca-file                         : <my organization root ca pem>
```

## Host Operating System
```bash
# Removed some information that may be sensitive
OS Name:                   Microsoft Windows 10 Enterprise
OS Version:                10.0.18363 N/A Build 18363
OS Manufacturer:           Microsoft Corporation
OS Configuration:          Member Workstation
OS Build Type:             Multiprocessor Free
Original Install Date:     8/13/2021, 6:20:59 PM
System Manufacturer:       Dell Inc.
System Model:              Precision 7550
System Type:               x64-based PC
Processor(s):              1 Processor(s) Installed.
                           [01]: Intel64 Family 6 Model 165 Stepping 2 GenuineIntel ~2712 Mhz
BIOS Version:              Dell Inc. 1.8.1, 7/19/2021
Windows Directory:         C:\windows
System Directory:          C:\windows\system32
Boot Device:               \Device\HarddiskVolume1
System Locale:             en-us;English (United States)
Input Locale:              en-us;English (United States)
Time Zone:                 (UTC+08:00) Kuala Lumpur, Singapore
Total Physical Memory:     65,165 MB
Available Physical Memory: 33,583 MB
Virtual Memory: Max Size:  130,701 MB
Virtual Memory: Available: 90,652 MB
Virtual Memory: In Use:    40,049 MB
Hyper-V Requirements:      A hypervisor has been detected. Features required for Hyper-V will not be displayed.
```

### Steps to reproduce

  1. Use default memory and CPU setting.
  2. Start CRC after configuration
  3. After I leave the machine for some time (machine does not enter sleep mode) and come back, I found high CPU usage although no activity by vmmem, and near 100% disk IO (which is > 2000MB/s on a NVME SSD) by System process.
  4. Adjusting the memory to 11G and above, no such issue. 10G - have such issue. 

### Expected
No such high CPU and Memory consumption

### Logs

Before gather the logs try following if that fix your issue
```bash
$ crc delete -f
$ crc cleanup
$ crc setup
$ crc start --log-level debug
```

Please consider posting the output of `crc start --log-level debug`  on http://gist.github.com/ and post the link in the issue.

**Comment by gbraad** [comment]
> After I leave the machine for some time

Idle with no load ? (no deployed application or containers)

**Comment by gbraad** [comment]
I did not see this same behavior. When idle I see an occasional spike on disk I/O; (HDD and SSD no difference); ThinkPad and ThinkCentre; fully updated.

Have this been observed on another machine or would it be possible to try this?

**Comment by charlesxucheng** [comment]
Thanks. I will get my colleague to try out on his laptop (a different model) and update here again.

**Comment by charlesxucheng** [comment]
Hello. I have gotten my colleague to try on his laptop (which has slower CPU clock but still an i7). He ran into startup issues with the default CPU and RAM settings. For example, "Cluster not ready: cluster operators are still not stable after 15m45s." So far he is unable to do login. Got errors like "Cannot update kubeconfig: read tcp 127.0.0.1:62567->127.0.0.1:6443: wsarecv: An existing connection was forcibly closed by the remote host." and "TLS handshake timeout". And the CPU usage is between 30% to 60% all the time.

**Comment by charlesxucheng** [comment]
Any updates on this?

**Comment by praveenkumar** [comment]
@charlesxucheng The issue which you observe from your colleague laptop is different and better to create a new issue using issue-template. During our testing on windows we didn't observe such behavior :( Since for you 11G is works as expected so better to set it with config and keep using it.
